lib/cartopy/tests/mpl/test_ticker.py

```diff
@@ -46,6 +46,17 @@ def test_LatitudeFormatter():
     assert result == expected
 
 
+def test_LatitudeFormatter_direction_label():
+    formatter = LatitudeFormatter(direction_label=False)
+    p = ccrs.PlateCarree()
+    formatter.axis = Mock(axes=Mock(GeoAxes, projection=p))
+    test_ticks = [-90, -60, -30, 0, 30, 60, 90]
+    result = [formatter(tick) for tick in test_ticks]
+    expected = ['90\u00B0', '60\u00B0', '30\u00B0', '0\u00B0',
```

Comment:
I would expect this to now include the negative signs instead of the W/E labels?
```suggestion
    expected = ['-90\u00B0', '-60\u00B0', '-30\u00B0', '0\u00B0',
```